include/items.php

```diff
@@ -401,17 +401,17 @@ function drop_item($id)
 		}
 		// Now check how the user responded to the confirmation query
 		if (!empty($_REQUEST['canceled'])) {
-			goaway(System::baseUrl() . '/' . $_SESSION['return_url']);
+			goaway('/item/drop/' . $id);
 		}
 
 		// delete the item
 		Item::deleteForUser(['id' => $item['id']], local_user());
 
-		goaway(System::baseUrl() . '/' . $_SESSION['return_url']);
+		goaway('/item/drop/' . $id);
```

Comment:
The item has been successfully deleted, we should redirect to the `/network` page since the item doesn't exist anymore.